cmd/admin-handlers.go

```diff
@@ -1951,3 +2135,77 @@ func checkConnection(endpointStr string, timeout time.Duration) error {
 	defer xhttp.DrainBody(resp.Body)
 	return nil
 }
+
+func createHostAnonymizerForFSMode() map[string]string {
+	hostAnonymizer := map[string]string{
+		globalLocalNodeName: "server1",
+	}
+
+	apiEndpoints := getAPIEndpoints()
+	for _, ep := range apiEndpoints {
+		if len(ep) > 0 {
+			if url, err := xnet.ParseHTTPURL(ep); err == nil {
+				// In FS mode the drive names don't include the host.
+				// So mapping just the host should be sufficient.
```

Comment:
Sure, but the error could maybe occur in other places where it doesn't indicate fs mode. We have global flags for erasure coded mode.

Reply:
This function is called based on the global flag only (`if !globalIsDistErasure`)